<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" name="Failed suite [Automation Test Suite for IX Onespace]">
  <listeners>
    <listener class-name="com.task.automation.util.RetryListener"/>
  </listeners>
  <test thread-count="5" name="Automation Test Cases(failed)">
    <classes>
      <class name="com.task.automation.test.SignIn">
        <methods>
          <include name="afterTest"/>
          <include name="endReport"/>
          <include name="config"/>
          <include name="beforeMethod"/>
          <include name="beforeTest"/>
          <include name="afterMethod"/>
          <include name="verifyLoginWithIncorrectCFailed"/>
        </methods>
      </class> <!-- com.task.automation.test.SignIn -->
    </classes>
  </test> <!-- Automation Test Cases(failed) -->
</suite> <!-- Failed suite [Automation Test Suite for IX Onespace] -->
